tabular/src/autogluon/tabular/models/catboost/catboost_model.py

```diff
@@ -12,15 +12,15 @@
 from .hyperparameters.searchspaces import get_default_searchspace
 from ..abstract.abstract_model import AbstractModel
 from ...constants import PROBLEM_TYPES_CLASSIFICATION, MULTICLASS, SOFTCLASS
+from ...features.feature_metadata import R_OBJECT
 from autogluon.core.utils.exceptions import NotEnoughMemoryError, TimeLimitExceeded
 from autogluon.core.utils import try_import_catboost, try_import_catboostdev
 
 logger = logging.getLogger(__name__)
 
 
-# TODO: Catboost crashes on multiclass problems where only two classes have significant member count.
-#  Question: Do we turn these into binary classification and then convert to multiclass output in Learner? This would make the most sense.
-# TODO: Consider having Catboost variant that converts all categoricals to numerical as done in RFModel, was showing improved results in some problems.
+# TODO: Consider having CatBoost variant that converts all categoricals to numerical as done in RFModel, was showing improved results in some problems.
+# TODO: v0.1 rename to CatBoostModel and rename model name default to CatBoost (instead of Catboost)
 class CatboostModel(AbstractModel):
     def __init__(self, **kwargs):
```

Comment:
I think it would be good if each model has some minimal documentation that at least states which dtypes will lead to a column being ignored by this model.  I don't think most users/contributors know about this key fact otherwise as it's buried deep in the code.

Reply:
added TODO